TypewiseAlert/FakeAlert.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace TypewiseAlert
+{
+    public class FakeAlert : IAlert, IFakeAlert
```

Comment:
What is the importance of IFakeAlert interface

Reply:
Removed the IFakeAlert interface